components/Home/HomeDownloadButton.tsx

```diff
@@ -15,8 +15,8 @@ const HomeDownloadButton = (props: HomeDownloadButtonProps) => {
   } = useNextraContext();
 
   const nodeDownloadLink = `https://nodejs.org/dist/${props.node}/`;
-  const nodeApiLink = `https://nodejs.org/dist/latest-${props.nodeMajor}/docs/api/`;
-
+  const nodeApiLink = `/latest-${props.nodeMajor}/docs/api/`;
```

Comment:
Are you sure this is correct?

Reply:
No I'm sure it's wrong - I'll fix it. I didn't mean to edit that line. Thanks